app/assets/stylesheets/baskets.css.scss

```diff
@@ -41,3 +41,10 @@
     margin-bottom: 0;
   }
 }
+
+.empty {
+  background: #EEE;
```

Comment:
Color literals like `#EEE` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#EEE` should be written as `#eee`